draft-bft-rats-kat.md

```diff
@@ -3,9 +3,8 @@ v: 3
 
 title: "Key Attestation Token"
```

Comment:
The Name Key Attestation itself is mis-leading

Reply:
> The Name Key Attestation itself is mis-leading

in what sense?

Reply:
We are not documenting Key Attestation here!

Reply:
😕